.github/workflows/pull-request.yml

```
@@ -410,4 +410,4 @@ jobs:
         if: github.event.pull_request.base.ref != 'develop-postgres'
         run: |
           echo "Error: Pull request target branch must be 'develop-postgres'. Please refer PR_GUIDELINES.md"
-          exit 1
+          exit 1
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)